sysutils/colima/Portfile

```diff
@@ -36,9 +40,18 @@ build.args          ./cmd/${name}
 
 destroot {
     xinstall -m 0755 ${worksrcpath}/${name} ${destroot}${prefix}/bin/
+
+    # install completion for Bash and Zsh
+    set bash_completions_dir ${destroot}${prefix}/share/bash-completion/completions
+    set zsh_completions_dir ${destroot}${prefix}/share/zsh/site-functions
+    xinstall -d ${bash_completions_dir} ${zsh_completions_dir}
+    system "${worksrcpath}/${name} completion bash > ${bash_completions_dir}/${name}"
+    system "${worksrcpath}/${name} completion zsh > ${zsh_completions_dir}/_${name}"
 }
 
 notes {
+    To run Docker containers locally, run 'colima start'.
```

Comment:
I am not in favour of this particular note, because while `colima start` works perfectly once I have my initial colima VM, I have to run a different command-line to initialize it or after delete / remove a Colima instance: `colima start --activate --memory 8 --vm-type vz --vz-rosetta`

If one prefers to use containerd, then `colima start --runtime containerd`. Using k8s? `colima start --kubernetes`. This is all documented on the Colima project page.

Reply:
Okay, I've dropped this change, for now. Just so you know, my thinking was:

1. `colima` uses `qemu` by default (even though you and I prefer `vz`) and that should be perfectly capable of serving up docker.
2. The default for `colima` is a `docker` runtime, indicating that this is the preferred option.

Personally, I hope they change the defaults to `vz` soon, so we can drop the dependency on `qemu`. It's just wasteful, IMO.